content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+{{< warning >}}
+The following information describes an experimental feature, which is intended
+for evaluation purposes only.
+{{< /warning >}}
+
+This task shows how to provision and manage DNS certificates in Istio.
```

Comment:
Can you explain why a user would want to do this? This seems more like an implementation detail to me than a task?

Reply:
Done: added the explanation.